cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java

```diff
@@ -168,9 +171,11 @@ public SparkExecutionContext create(SparkRuntimeContext runtimeContext) {
         File expandedProgramJar = Locations.linkOrCopy(runtimeContext.getProgram().getJarLocation(),
                                                        new File(tempDir,
                                                                 SparkRuntimeContextProvider.PROGRAM_JAR_EXPANDED_NAME));
-        // Localize both the unexpanded and expanded jar
-        localizeResources.add(new LocalizeResource(programJar, false));
+        // Localize both the unexpanded and expanded jar program jar
+        localizeResources.add(new LocalizeResource(programJar));
```

Comment:
this is an archive right? Shouldn't it be ``new LocalizeResource(programJar, true)``?